### PR TITLE
Add a CONTRIBUTING guide

### PR DESCRIPTION
**Pull request by jonny-improbable**
Give instructions for how to get the local testing environment setup as it's not the easiest project to work with ;)

Whilst I was at it I had a debug a nightmare of an issue in order to get the latest version of Chrome to run the tests over https; this required changes to the certificate configuration to include the full list of `subjectAltName` covering all of the hostnames that the tests try to connect to when they execute.

More context can be found [here](https://stackoverflow.com/questions/46349459/chrome-neterr-cert-authority-invalid-error-on-self-signing-certificate-at-loca) if you're interested :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* [dep](https://golang.github.io/dep/docs/installation.html) - Go dependency manager
+* [nvm](https://github.com/creationix/nvm#installation) - Node Version Manager (for installing NodeJS and NPM)
+
+## Performing a Fresh Checkout
+The following steps guide you through a fresh checkout
+
+```
+# Initalise a Go Workspace and checkout project sources
+go get -u github.com/improbable-eng/grpc-web/go
+cd src/github.com/improbable-eng/grpc-web
```
Comment (johanbrandhorst) [inline CONTRIBUTING.md]: This command is relative to `$GOPATH`, do we want to make it a little more explicit?

```
cd $GOPATH/src/...
```

**Comment by johanbrandhorst** [comment]
Thanks for going through this, it'll be a great addition!

File: CONTRIBUTING.md (hunk 1 above)
Comment (jonny-improbable) [reply CONTRIBUTING.md]: Better yet, I should probably set the `GOPATH` :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* [dep](https://golang.github.io/dep/docs/installation.html) - Go dependency manager
+* [nvm](https://github.com/creationix/nvm#installation) - Node Version Manager (for installing NodeJS and NPM)
+
+## Performing a Fresh Checkout
+The following steps guide you through a fresh checkout
+
+```
+# Create a workspace
+cd ~/Projects/grpc-web  # or wherever you want your checkout to live
+export GOPATH=`pwd`
```
Comment (johanbrandhorst) [inline CONTRIBUTING.md]: I prefer `$()`, I think backtick notation is only used for compatibility, right?

**Pull request merged**